### ISSUE TITLE
Deployment won't run after server upgrade

### ISSUE THREAD

**Issue by Phoenix997733** [vsys-host/shkeeper.io#50]
Hey,

I had shkeeper working on my server but because tron blockchain is continuously increasing in space I ran out of space and the gateway stopped working. After making a server upgrade, it's still not working. In get pods, the java-tron and other pods seems most are working but shkeeper is not also cred-generator is not. I tried:
- restarted the pods but also wasn't able to until reached timeout
- rebooted the system but didn't solve the problem
- helm repo update and kubectl upgrade to values also give me that it deployed the new values but still can't reach the gateway through port 5000
- give it some time maybe it needs to download the blockchain but also after 12 hours nothing and network traffic seems to be normal now (there was a lot of traffic when I first upgraded the server)

Is there any solution to reset the installation or the pods or any solution to get it to work without losing data and wallets?

Thank you.

**Comment by yukonet** [comment]
Try to check shkeeper log with a command: `kubectl -n shkeeper logs <your_shkeeper_pod_name>`

**Comment by Phoenix997733** [comment]
Thank you so much for your reply. After writing the issue I tried many things. I don't know if it contributes to the problem solution:
I deleted all Completed and error pods
I checked the values.yaml file and modified things in it
I think there was a problem with the number of the version written in it so I modified it based on the helm charts values file.
The gateway is now working and accessible but still has a problem with enabling Ethereum and BNB on public node.
The problem of the shkeeper pod is still there also. I'll attach the log file in the next comment but it seems that there's a problem with the secret key.
